roles/iiif/README.md

```diff
@@ -1,22 +1,108 @@
-Role Name
-=========
+# Role Name: iiif
 
-Installs [Samvera Labs](https://github.com/samvera-labs/serverless-iiif)
+Deploys [Samvera Labs serverless-iiif](https://github.com/samvera-labs/serverless-iiif) - a IIIF Image API 2.1/3.0 server running on AWS Lambda.
```

Comment:
I'd love to see a description here of what serverless-iiif does in layperson's terms - mostly because I don't entirely understand it. Best I can come up with right now is: 
"Serverless-iiif supports serving images and derivatives from cloud buckets to URLs within our applications using serverless computing platforms in the cloud. For example, images from libimages2 are now stored in an S3 bucket and served by an AWS Lambda."